### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by Arunsai14**
## Description

Please provide a brief description of the changes introduced by this pull request.

## Related Issue

- [Issue Number](link-to-issue): Brief description

## Checklist

Please ensure that the following steps are completed before submitting the pull request:

- [ ] Code follows the Terraform best practices and style guidelines.
- [x] Changes are appropriately documented, including any necessary updates to README or other documentation files.
- [ ] Unit tests have been added or updated to cover the changes introduced by this pull request.
- [ ] Changes have been tested locally and verified to work as expected.
- [ ] The code has been reviewed to ensure it aligns with the project's goals and standards.
- [ ] Dependencies and backward compatibility have been considered and addressed if applicable.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Testing Instructions

Provide detailed steps or instructions for testing the changes introduced by this pull request.

## Screenshots

Include any relevant screenshots or visual aids to help reviewers understand the changes visually, if applicable.

## Additional Notes

Add any additional notes or context that might be helpful for reviewers or users testing the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,50 +123,3 @@ module "cloud_security" {
 | <a name="output_security_hub_sns_topic"></a> [security\_hub\_sns\_topic](#output\_security\_hub\_sns\_topic) | The SNS topic that was created |
 | <a name="output_security_hub_sns_topic_subscriptions"></a> [security\_hub\_sns\_topic\_subscriptions](#output\_security\_hub\_sns\_topic\_subscriptions) | The SNS topic that was created |
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-
-### Git commits
```
Comment (mayank0202) [inline README.md:125]: why this all stuff is removed from root file i.e. README.md?

cc: @tsaucier-sf  @jamescrowley321 @vijay-stephen

**Review by mayank0202 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -14,7 +16,7 @@ To see a full example, check out the [main.tf](./example/main.tf) file in the ex
 ```hcl
 module "cloud_security" {
   source      = "sourcefuse/arc-security/aws"
-  version     = "0.0.4"
```
Comment (jamescrowley321) [inline README.md:17]: We can skip the version in the readme as it is difficult to maintain.

File: README.md (hunk 1 above)
Comment (jamescrowley321) [reply README.md:125]: Why is this content removed?

**Review by jamescrowley321 — APPROVED** [review]

**Review by tsaucier-sf — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Arunsai14) [reply README.md]: reverted

**Review by mayank0202 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mayank0202) [reply README.md:17]: yes please remove the `version` from here as by default it picks up the latest version from registry

**Review by jamescrowley321 — APPROVED** [review]

**Pull request merged**